Website/Src/Startup.Container.cs

```diff
@@ -187,6 +188,21 @@ public void ConfigureServices(IServiceCollection services)
             // Allow IOptions<T> to be available through DI
             services.AddOptions();
 
+            services.AddSwaggerGen(options =>
+            {
+                options.SwaggerDoc(
+                    "v1",
```

Comment:
Is this required? At least for now we don't really have a versioning story (or plan to)

Reply:
Not required, let me check up on all the options of this to have what we want.